File: 2023/domaci/02-trivialni-rovnice/volfmatous/main.js
const readline = require("readline");
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout,
});

let lines = 0;
let a, b, c;

rl.on("line", (line) => {
    lines++;
    switch (lines) {
        case 1:
            a = parseFloat(line);
            return;

        case 2:
            b = parseFloat(line);
            return;
    }

    c = parseFloat(line);

    rl.close();

    let output;

    if (b === 1) {
        output = "undefined";
    } else {
        output = (c - a) / (b + 1);
    }

    console.log(output);
});

File: 2023/domaci/02-trivialni-rovnice/volfmatous/main.py
if __name__ == "__main__":
    a = float(input())
    b = float(input())
    c = float(input())

    if b == 1:
        output = "undefined"
    else:
        x = (c - a) / (b + 1)
        output = f"{x:g}"

    print(output)
